pittsburgh/__init__.py

```diff
@@ -0,0 +1,81 @@
+# encoding=utf-8
+from pupa.scrape import Jurisdiction, Organization, Person
+
+from .events import PittsburghEventsScraper
+from .people import PittsburghPersonScraper
+from .bills import PittsburghBillScraper
+
+import datetime
+
+
+class Pittsburgh(Jurisdiction):
+    division_id = "ocd-division/country:us/state:pa/place:pittsburgh"
+    classification = "government"
+    name = "Pittsburgh City Government"
+    timezone = "America/New_York"
+    url = "https://pittsburgh.legistar.com"
+
+    scrapers = {
+        "events": PittsburghEventsScraper,
+        "people": PittsburghPersonScraper,
+        "bills": PittsburghBillScraper,
+    }
+
+    legislative_sessions = []
+
+    def get_legislative_sessions(self):
+        for year in range(2001, 2030):
+            legislative_sessions.append({"identifier": str(year),
+                                         "name": str(year) + " Session",
+                                         "start_date": str(year) + "-01-01",
+                                         "end_date": str(year) + "-12-31"})
+
+    def get_organizations(self):
+        org = Organization(name="Pittsburgh City Council", classification="legislature")
+        for x in range(1, 10):
+            org.add_post(
+                "District {}".format(x),
+                "Councilmember",
+                division_id="ocd-division/country:us/state:pa/place:pittsburgh/council_district:{}".format(x))
+        yield org
+
+        standing_committee = Organization(name="Standing Committee", classification="committee")
+        standing_committee.add_source("http://pittsburghpa.gov/council/standing-committees", note="web")
+        yield standing_committee
+
+        mayor = Organization(name="Mayor", classification="executive")
+        mayor.add_post("Mayor", "Mayor", division_id="ocd-division/country:us/state:pa/place:pittsburgh")
+        mayor.add_source("http://pittsburghpa.gov/mayor/index.html", note="web")
+        yield mayor
```

Comment:
One thing I wasn't sure about with the `Mayor` org: Our last two mayors (William Peduto and Luke Ravenstahl) were council members before being elected, so the scraper throws an error asking them to be disambiguated if I register them with their mayoral terms/posts in this file. Maybe that's fine?

Reply:
Addin their birth year should do it.

Reply:
Unfortunately still getting errors. If I do the following:

```
        peduto.add_term("Mayor",
                        "executive",
                        start_date=datetime.date(2014, 1 ,6),
                        birth_date=datetime.date(1964, 10, 30),
                        appointment=True)
```
The import fails with `TypeError: add_term() got an unexpected keyword argument 'birth_date'`.

When I try `peduto = Person(name="William Peduto", birth_date=datetime.date(1964, 10, 30))`, I get `pupa.exceptions.SameNameError: multiple people with same name "William Peduto" in Jurisdiction - must provide birth_date to disambiguate`

Is there another way of adding `birth_date` I'm missing?